Update vipps-login-api-howitworks.md (#16)
vippsas/vipps-login-api: vipps-login-api-howitworks.md

### 2. The Vipps landing page

> If the login was started on a desktop device the user will be sent to the Vipps landing page. If the user is not remembered in browser from earlier, the user enters her phone number, and can choose to be remembered in the browser for a quicker login the next time. The "Remember me in browser" option applies to the browser, independent of merchants. This option is default off on desktop devices.

Reviewer: I agree with having more information about browser recognition here, but I think the wording is a bit unclear.
How about:
```suggestion
If the login was started on a desktop device the user will be sent to the Vipps landing page if the user has not previously opted in to having her browser remembered. The user enters her phone number, and can choose to be remembered in the browser for a quicker login the next time. The "Remember me in browser" option applies to the browser, independent of merchants. This option defaults to off for desktop devices.
```

The user is then prompted to log in to Vipps, and follow the instructions in the app.

> If the login was started from a mobile device and the user needs to authenticate in the app, the user will confirm to be taken  to the Vipps-app, without having to enter her phone number.

Reviewer: ```suggestion
If the login was started from a mobile device and the user needs to authenticate in the app, the user will confirm to be taken  to the Vipps-app without having to enter her phone number.
```

![Vipps landing page](images/vipps-login-step2.svg)

### 3. E-mail verification (first time only)

If the merchant has requested the user’s e-mail, the user have to verify their e-mail address in Vipps the first time they use Vipps login or if they change their e-mail at a later time. The user can also check and change their registered addresses. This is only necessary to do once. If the merchant has not requested the user's e-mail, this step is skipped.

![Accept agreement](images/vipps-login-step3.svg)

### 4. Confirm login

The user confirms the login in the Vipps-app.

> If the login was started on a phone, the user can in this step choose to be remembered in the browser for a quicker registration or login the next time. The "Remember me in browser" option is default on for mobile devices.

Reviewer: ```suggestion
If the login was started on a phone, the user can in this step choose to be remembered in the browser for a quicker registration or login the next time. The "Remember me in browser" option defaults to on for mobile devices.
```
